package.json

```diff
@@ -50,10 +44,9 @@
     "webpack-dev-server": "1.16.3"
   },
   "scripts": {
-    "organizer-dev": "NODE_ENV=debug webpack-dev-server --content-base build/organizer --port 8080",
-    "attendee-dev": "NODE_ENV=debug webpack-dev-server --content-base build/attendee --port 8081",
-    "build-prod": "NODE_ENV=prod webpack",
-    "build-stage": "NODE_ENV=staging webpack"
+    "organizer-dev": "webpack-dev-server --content-base build/organizer --port 8080",
+    "attendee-dev": "webpack-dev-server --content-base build/attendee --port 8081",
+    "build": "NODE_ENV=production webpack"
```

Comment:
no, every combination of minification + sourcemaps ended up being bigger than just neither. We'll use prod for both dev.calligre and symposium.calligre